index.php

```diff
@@ -75,29 +75,12 @@
-				<?php
-			}elseif (isset($_COOKIE["engine"]) && $_COOKIE["engine"] == "exalead" && $lang=="de") {
-				?>
-				<div id="form"><form id="frmSearch" action="go" method="get" accept-charset="UTF-8"><input type="text" id="txtSearch" name="q" alt="Search Criteria" onkeyup="<?php print htmlspecialchars($onKeyUp); ?>" autocomplete="off" /><input type="hidden" name="l" value="<?= $lang ?>" /><input type="hidden" name="e" value="exalead" /><input type="hidden" name="s" value="suchen" />&nbsp; <input type="submit" id="cmdSearch" name="b" value="<?= myText("search") ?>" alt="Suche starten" /><img src="img/exalead.ico" width="16" height="16" title="Suchen mit exalead" /></form></div>
-				<?php
-			}elseif (isset($_COOKIE["engine"]) && $_COOKIE["engine"] == "web.de" && $lang=="de") {
-				?>
-				<div id="form"><form id="frmSearch" action="go" method="get" accept-charset="UTF-8"><input type="text" id="txtSearch" name="q" alt="Search Criteria" onkeyup="<?php print htmlspecialchars($onKeyUp); ?>" autocomplete="off" /><input type="hidden" name="l" value="<?= $lang ?>" /><input type="hidden" name="e" value="web.de" /><input type="hidden" name="s" value="suchen" />&nbsp; <input type="submit" id="cmdSearch" name="b" value="<?= myText("search") ?>" alt="Suche starten" /><img src="img/web.de.ico" width="16" height="16" title="Suchen mit web.de" /></form></div>
-				<?php
-			} else {
-				?>
-				<div id="form"><form id="frmSearch" action="go" method="get" accept-charset="UTF-8"><input type="text" id="txtSearch" name="q" alt="Search Criteria" onkeyup="<?php print htmlspecialchars($onKeyUp); ?>" autocomplete="off" /><input type="hidden" name="l" value="<?= $lang ?>" /><input type="hidden" name="e" value="wikipedia" /><input type="hidden" name="s" value="suchen" />&nbsp; <input type="submit" id="cmdSearch" name="b" value="<?= myText("search") ?>" alt="Suche starten" /></form></div>
-				<?php
-			}
-			?>
+			<h3><?php echo myText("searchin") ?></h3>
+			<div id="form"><form id="frmSearch" action="go.php" method="get" accept-charset="UTF-8"><input type="text" id="txtSearch" name="q" alt="Search Criteria" onkeyup="<?php print htmlspecialchars($onKeyUp); ?>" autocomplete="off" /><input type="hidden" name="l" value="<?php echo $lang ?>" /><input type="hidden" name="e" value="wikipedia" /><input type="hidden" name="s" value="suchen" />&nbsp; <input type="submit" id="cmdSearch" name="b" value="<?php echo myText("search") ?>" alt="Suche starten" /></form></div>
```

Comment:
Ugh, I remember `go` being rewritten to `go.php` by the web server. Maybe this was meant to look better in the URL. Maybe it just didn't because of all those single-character variable names.

Reply:
I changed it because it breaks on Docker and ´go.php´ is never really visible to users unless the look at the little URL preview that some browsers offer on hover over search result elements. Is this a problem?